bundle/src/main/java/com/adobe/acs/commons/workflow/bulk/execution/impl/Status.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2016 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package com.adobe.acs.commons.workflow.bulk.execution.impl;
```

Comment:
and, in fact, running a build shows this:

    [WARNING] Bundle com.adobe.acs:acs-aem-commons-bundle:bundle:2.5.1-SNAPSHOT : Export com.adobe.acs.commons.workflow.bulk.execution,  has 1,  private references [com.adobe.acs.commons.workflow.bulk.execution.impl],
    [WARNING] Bundle com.adobe.acs:acs-aem-commons-bundle:bundle:2.5.1-SNAPSHOT : Export com.adobe.acs.commons.workflow.bulk.execution.model,  has 1,  private references [com.adobe.acs.commons.workflow.bulk.execution.impl],